### PR TITLE
ripdrag: make gtk theming function properly by adding wrapGAppsHook

### PR DESCRIPTION
**Pull request by ArtixBTW**
This is needed for gtk theming to function properly. (Picture below is before vs after)
20230724_22h02m24s_grim

###### Description of changes
Just adding `wrapGAppsHook` to `nativeBuildInputs`

###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) (or backporting [23.05 Release notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2305.section.md))
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/tools/misc/ripdrag/default.nix [hunk 1]
```diff
@@ -1,4 +1,4 @@
-{ lib, rustPlatform, fetchCrate, pkg-config, gtk4 }:
+{ lib, rustPlatform, fetchCrate, pkg-config, wrapGAppsHook, gtk4 }:
```
Comment (figsoda) [inline pkgs/tools/misc/ripdrag/default.nix:1]:
```suggestion
{ lib, rustPlatform, fetchCrate, pkg-config, wrapGAppsHook4, gtk4 }:
```

File: pkgs/tools/misc/ripdrag/default.nix [hunk 2]
```diff
@@ -11,7 +11,7 @@ rustPlatform.buildRustPackage rec {
 
   cargoSha256 = "sha256-C2I26E/dd18A4DDgOYGR8aS1RBrrNUwaXI4ZJHcrKy0=";
 
-  nativeBuildInputs = [ pkg-config ];
+  nativeBuildInputs = [ pkg-config wrapGAppsHook ];
```
Comment (figsoda) [inline pkgs/tools/misc/ripdrag/default.nix:14]:
```suggestion
  nativeBuildInputs = [ pkg-config wrapGAppsHook4 ];
```

**Review by figsoda — COMMENTED** [review]
this might be a little better since we are using gtk4

**Comment by figsoda** [comment]
included in #245406

**Comment by ArchB1W** [comment]
Cool! Guess I'll close this then.

**Pull request closed without merging**